app/services/client/visibility.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const $window = window,
+  $document = document,
+  _throttle = require('lodash/throttle'),
+  Eventify = require('eventify');
+var list = [],
```

Comment:
Remember to avoid using `var`, the recommended are `const` for constant properties and `let` for everything else